charts/ocis/templates/_common/registry/registry.tpl

```diff
@@ -2,12 +2,21 @@
 oCIS service registry
 */}}
 {{- define "ocis.serviceRegistry" -}}
-{{- $valid := list "kubernetes" "etcd" "natsjs" -}}
+{{- $valid := list "etcd" "nats-js-kv" -}}
```

Comment:
since we no longer support "kubernetes" as a registry, we need to also get rid of the ServiceAccount for it, see `charts/ocis/templates/_common/registry/kubernetes`

Reply:
hopefully everything removed